[2021-09-27 14:07:46,511] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_gcs_result2 2021-09-27T14:06:38.240034+00:00 [queued]>
[2021-09-27 14:07:46,547] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_gcs_result2 2021-09-27T14:06:38.240034+00:00 [queued]>
[2021-09-27 14:07:46,548] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:07:46,550] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 14:07:46,552] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:07:46,568] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): create_entry_gcs_result2> on 2021-09-27T14:06:38.240034+00:00
[2021-09-27 14:07:46,572] {standard_task_runner.py:52} INFO - Started process 1375 to run task
[2021-09-27 14:07:46,577] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'create_entry_gcs_result2', '2021-09-27T14:06:38.240034+00:00', '--job-id', '14', '--pool', 'default_pool', '--raw', '--subdir', '/home/***/.local/lib/python3.6/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpyqticewz', '--error-file', '/tmp/tmpf2l02oyn']
[2021-09-27 14:07:46,580] {standard_task_runner.py:77} INFO - Job 14: Subtask create_entry_gcs_result2
[2021-09-27 14:07:46,687] {logging_mixin.py:109} INFO - Running <TaskInstance: example_complex.create_entry_gcs_result2 2021-09-27T14:06:38.240034+00:00 [running]> on host fa8cc667bc0b
[2021-09-27 14:07:46,826] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_complex
AIRFLOW_CTX_TASK_ID=create_entry_gcs_result2
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T14:06:38.240034+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-27T14:06:38.240034+00:00
[2021-09-27 14:07:46,834] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-27 14:07:46,849] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo create_entry_gcs_result2']
[2021-09-27 14:07:46,881] {subprocess.py:74} INFO - Output:
[2021-09-27 14:07:46,908] {subprocess.py:78} INFO - create_entry_gcs_result2
[2021-09-27 14:07:46,910] {subprocess.py:82} INFO - Command exited with return code 0
[2021-09-27 14:07:46,994] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=create_entry_gcs_result2, execution_date=20210927T140638, start_date=20210927T140746, end_date=20210927T140746
[2021-09-27 14:07:47,148] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 14:07:47,298] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
